agreements/bootstrapping/2024-05-18-bootstrapping-sdkman.md

```diff
@@ -0,0 +1,79 @@
+---
+status: draft
+---
```

Comment:
```suggestion
```

Reply:
you win. I should have done it this way. 🤦‍♀️ 🤣

Reply:
Hah, had mostly hoped it would be a one-click apply from here, but it seems I'm not able to edit the pull request files. So for now it lowers the barrier at least to drop this draft status.

Reply:
I saw this but just copied what all the others did.
Happy to remove it from the rest too in this PR if you like.